### PR TITLE
👌 IMPROVE: create_function depreciation

### PR DESCRIPTION
**Pull request by wonder32**
as of php 7.2 create_function() is deprecated
replaced it with a nameless function

### CODE REVIEW

File: class-wposa.php [hunk 1]
```diff
@@ -200,7 +200,9 @@ function admin_init() {
 	            $section['desc'] = '<div class="inside">' . $section['desc'] . '</div>';
 
 	            // Create the callback for description.
-	            $callback = create_function( '', 'echo "' . str_replace( '"', '\"', $section['desc'] ) . '";' );
+	            $callback = function() use ($section) {
```
Comment (ahmadawais) [inline class-wposa.php:203]: `function() use` what kind of syntax is that? Is it PHP 5.4 compatible?

Comment (ahmadawais) [reply class-wposa.php:203]: Also let me know how did you test this PR?

**Review by ahmadawais — COMMENTED** [review]
Looking forward to some explanation on how you have tested this PR and the backward compatibility.

File: class-wposa.php (hunk 1 above)
Comment (wonder32) [reply class-wposa.php:203]: I do not have php 5.4 anymore, tested on 5.6, 7.0, 7.2
I use it in 

[woocommerce-notification](https://github.com/wonder32/woocommerce-notification) | [file](https://github.com/wonder32/woocommerce-notification/blob/master/Includes/Settings.php)

syntax example [link](http://php.net/manual/en/functions.anonymous.php#example-165)

**Comment by ahmadawais** [comment]
Merged! Thanks @wonder32  and welcome to the project as a contributor.

**Comment by ahmadawais** [comment]
Merged! Thanks @wonder32  and welcome to the project as a contributor.

**Pull request merged**